paper.md

```diff
@@ -76,19 +76,21 @@ Finally, as both data management plan and data protection impact assessment are
 
-*TODO: (Pinar), Vilém, Nene, and Paulette - what was the process, how you worked, what is the result*
+The DPIA knowledge model is mainly inspired by the several existing documents, namely the [CNIL DPIA document template](https://www.cnil.fr/sites/default/files/atoms/files/cnil-pia-2-en-templates.pdf), the DPIA document template used by the Luxembourg Centre for Systems Biomedicine and [ICO guidance](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/data-protection-impact-assessments-dpias/how-do-we-do-a-dpia/#how)) on data protection.
 
-The DPIA knowledge model aims to help in managing the data protection risks of a research project by identifying and analyzing those risks and the solutions implemented to mitigate them. This knowledge model is mainly inspired by the [CNIL Template](https://www.cnil.fr/sites/default/files/atoms/files/cnil-pia-2-en-templates.pdf), the DPIA template from the Luxembourg Centre for Systems Biomedicine (LCSB) DPIA Template, and some [ICO guidance](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/data-protection-impact-assessments-dpias/how-do-we-do-a-dpia/#how)). The DPIA knowledge model describes:
+The first chapter collects administrative information and description of method followed for during project's DPIA. Following chapters contain questions related to:
 
+* The identification of the need for a DPIA
+* The nature, scope, context and purposes of the processing of data, as well as the list of related responsibilities
+* The data processes and supporting assets
+* The proportionality and necessity of the processing
+* The measures taken to protect the rights of the subjects
+* The identification, analysis, and evaluation of the data risks factors and the measures and controls to mitigate those risks
 
-* The identification of the need for a DPIA;
-* The nature, scope, context and purposes of the processing of data, as well as the list of related responsibilities;
-* The data processes and supporting assets;
-* The proportionality and necessity of the processing;
-* The measures taken to protect the rights of the subjects;
-* The identification, analyzis, and evaluation of the data risks factors and the measures and controls to mitigate those risks.
+Although a risk assessment is very important part of a DPIA, our knowledge model is addressing it very briefly only by questions on general controls which are or will be in place. As risks generally depend on the used infrastructure and involved institutional units, they are globally applicable to all the processing under same institute. We propose process, where the risk assessment matrix can be generated by in-house or external professionals using dedicated tools (e.g. [Monarc](https://www.monarc.lu/)) and reused for each DPIA as a resource. For this purpose, our knowledge model includes question to provide link to such a document.
```

Comment:
Although a risk assessment is **a** very important part of **the** DPIA

Reply:
our knowledge model is addressing it very briefly by questions on general controls

Reply:
As risks generally depend on the infrastructure used and the  institutional units involved, they would apply globally to all the processing under their authority.

Reply:
We propose a process where the risk assessment matrix can be generated by in-house or external professionals using dedicated tools (e.g. [Monarc](https://www.monarc.lu/)) and reused for each DPIA as a resource.

Reply:
For this purpose, our knowledge model includes **a** question to provide link to such a document.